examples/mysql-wordpress-pd/README.md

```diff
@@ -336,27 +266,33 @@ http://<external-ip>
 
 You should see the familiar WordPress init page.
 
+![WordPress init page](WordPress.png "WordPress init page")
+
```

Comment:
Add the following warning message as we did in https://cloud.google.com/container-engine/docs/tutorials/persistent-disk/#visit_your_new_wordpress_blog?

> 
Warning: Do not leave your WordPress installation on this page. If it is found by another user, they can set up a website on your instance and use it to serve potentially malicious content. You should either continue with the installation past the point at which you create your username and password, delete your instance, or set up a firewall to restrict access.